uint32 max_distance
---
bool success

